python/Ganga/new_tests/GPI/newXMLTest/TestXMLCorruption.py

```diff
@@ -0,0 +1,105 @@
+from __future__ import absolute_import
+
+from ..GangaUnitTest import GangaUnitTest
+
+from os import path, stat, unlink
+
+import time
+
+from .utilFunctions import getJobsPath, getXMLDir, getXMLFile, getIndexFile
+
+global_AutoStartReg = True
+
+notXMLStr = ['ThisIsNOTXML' for _ in range(20)]
+badStr = ''.join(notXMLStr)
+
+class TestXMLGenAndLoad(GangaUnitTest):
+
+    def setUp(self):
+        """Make sure that the Job object isn't destroyed between tests"""
+        super(TestXMLGenAndLoad, self).setUp()
+        from Ganga.Utility.Config import setConfigOption
+        setConfigOption('TestingFramework', 'AutoCleanup', 'False')
+        setConfigOption('Configuration', 'AutoStartReg', global_AutoStartReg)
+
+    def test_a_JobConstruction(self):
+        """ First construct the Job object (singular)"""
+        from Ganga.Utility.Config import getConfig
+        self.assertFalse(getConfig('TestingFramework')['AutoCleanup'])
+
+        from Ganga.GPI import Job, jobs
+        j=Job()
+        assert len(jobs) == 1
+
+        global global_AutoStartReg
+        global_AutoStartReg = False
```

Comment:
Given the object shouldn't ever exist out of the scope of the test I'm leaving it as a global